<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bilibili Web Crawler &amp;&amp; Wordcloud</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="video_bv_lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>video_bv若不填则默认获取上次结果</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wordcloud_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>wordcloud_name默认为wordcloud.png</string>
    </property>
   </widget>
   <widget class="QLabel" name="video_bv_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>视频的BV号</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordcloudAttribute">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>词云图属性</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_wordcloud_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>生成</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordcloud_name_label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>    词云图名字
(默认后缀为.png)</string>
    </property>
   </widget>
   <widget class="QLabel" name="crawlerSettings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>爬虫设置</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_pages_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>爬取评论区页数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_pages_lineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>max_pages默认为10(不宜过大)</string>
    </property>
   </widget>
   <widget class="QLabel" name="wordcloud_path_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>词云图存放路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wordcloud_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>wordcloud_path默认为当前文件夹下</string>
    </property>
   </widget>
   <widget class="QLabel" name="mask_pic_path_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>词云图形状
(蒙版路径)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mask_pic_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>mask_pic_path默认为./mask_pic/</string>
    </property>
   </widget>
   <widget class="QLabel" name="mask_pic_name_label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>       词云图形状
(蒙版图片名称+后缀)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mask_pic_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>mask_pic_name默认为1.png</string>
    </property>
   </widget>
   <widget class="QLabel" name="font_path_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字体路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="font_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>font_path默认为./font/simfang.ttf</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_color_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>背景颜色</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="background_color_lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>background_color默认为white</string>
    </property>
   </widget>
   <widget class="QLabel" name="width_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>宽</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="width_lineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>width默认为800</string>
    </property>
   </widget>
   <widget class="QLabel" name="height_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>340</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>长</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="height_lineEdit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>height默认为600</string>
    </property>
   </widget>
   <widget class="QLabel" name="crab_pic">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>420</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/crab.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bilibili_logo_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/bilibili_logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="about">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Made by 陈怀宇</string>
    </property>
   </widget>
   <widget class="QLabel" name="about2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(24062013)</string>
    </property>
   </widget>
   <widget class="QLabel" name="about_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/about.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pachong_pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>爬取评论</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="del_checkBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>350</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分词结果过滤</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cookies_lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>若成功登录则无需填写</string>
    </property>
   </widget>
   <widget class="QLabel" name="cookies_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>cookies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>不知道怎么获取cookies?
点我</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
